vector/build.gradle

```diff
@@ -167,6 +167,7 @@ dependencies {
 
     // UI
     implementation 'com.android.support.constraint:constraint-layout:1.1.0'
+    implementation 'com.getbase:floatingactionbutton:1.10.1'
```

Comment:
Can you credit this library in the file `open_source_licenses.html` please? There is already a section for Apache 2.0 license

Reply:
I have credited the library as requested.

Reply:
Thanks